coldcard-cli/src/main.rs

```diff
@@ -782,6 +824,69 @@ fn handle(cli: Cli) -> Result<(), Error> {
             }
             println!("{}", xpub);
         }
+        Command::MiniscriptList => {
+            let resp = cc.miniscript_list()?;
+            for m in resp {
+                println!("{m}");
+            }
+        }
+        Command::MiniscriptDelete { name } => {
+            let descriptor_name = match DescriptorName::new(name.clone()) {
+                Ok(n) => n,
+                Err(e) => {
+                    println!("ERROR: Invalid miniscript descriptor name: {e:?}.");
+                    return Ok(());
+                }
+            };
+            if let Err(e) = cc.delete_miniscript(descriptor_name) {
+                println!("ERROR: Fail to delete descriptor {name}: {e:?}")
+            }
+        }
+        Command::MiniscriptPolicy { name } => {
+            let descriptor_name = match DescriptorName::new(name.clone()) {
+                Ok(n) => n,
+                Err(e) => {
+                    println!("ERROR: Invalid miniscript descriptor name: {e:?}.");
+                    return Ok(());
+                }
+            };
+            match cc.bip388_policy_get(descriptor_name) {
+                Ok(Some(p)) => println!("{p}"),
+                Err(e) => {
+                    println!("ERROR: Fail to get miniscript policy for name {name}: {e:?}")
+                }
+                _ => {
+                    println!("ERROR: No descriptor found with name {name}");
+                }
+            }
+        }
+        Command::Restore {
+            password,
+            path,
+            plaintext,
+            tmp,
+        } => {
+            if !path.exists() {
+                println!("ERROR: Path {} does not exists", path.to_str().unwrap());
+                return Ok(());
+            }
+            if !path.is_file() {
+                println!("ERROR: Path {} is not a file", path.to_str().unwrap());
+                return Ok(());
+            }
+            let mut file = match File::open(path.clone()) {
+                Ok(f) => f,
+                Err(e) => {
+                    println!("ERROR: Fail to open file {}: {e}", path.to_str().unwrap());
```

Comment:
Errors and other output for human consumption is written out using `eprintln` (stderr). `println` (stdout) is for machines so that we can pipe between processes (e.g. [this one](https://github.com/alfred-hodler/rust-coldcard/pull/17/commits/22654621edb79418590eb7bc1c94929da26c16a9#diff-d4e58b3d40e20badb16dcd9eb55dc4116927fc94bba4fec73fb3619599df178bR830) is fine.